### PR TITLE
Add cleaning info for Raven Rock Enhanced

### PR DESCRIPTION
**Pull request by cecell**
Added cleaning info for Raven Rock Enhanced

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -16672,3 +16672,15 @@ plugins:
       - <<: *alreadyInX
         subs: [ 'CCOR_Vokrii.esp']
         condition: 'active("CCOR_Vokrii.esp") and active("Vokrii - WACCF Patch.esp")'
+
+  ### Raven Rock Enhanced ###
+  - name: 'RREnhanced.esp'
+    dirty:
```
Comment (MacSplody) [inline masterlist.yaml]: That's fine, I'd add a link just for future updates.
```suggestion
  - name: 'RREnhanced.esp'
    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/19001' ]
    dirty:
```

**Comment by cecell** [comment]
What do I need to do? Looks like another commit posted already

**Review by MacSplody — APPROVED** [review]

**Comment by MacSplody** [comment]
@cecell Thanks.

> What do I need to do? Looks like another commit posted already

Nothing if you apply a suggestion, it adds a new commit to the pull.

**Comment by cecell** [comment]
Outstanding thanks!

**Pull request merged**